mail/tasks.py

```diff
@@ -18,19 +18,17 @@ def _get_recipients_for_video(video):
     """
     Returns a list of recipients that will receive notifications for a video.
 
-    NOTE: this needs to be updated to include all the admin moira lists associated to the collection
-    where the video is.
-
     Args:
         video (ui.models.Video): a video object
 
     Returns:
         list: a list of strings representing emails
     """
-    recipients_list = []
-    owner_email = video.collection.owner.email
-    if owner_email:
-        recipients_list.append(owner_email)
+    admin_lists = [mlist.name for mlist in video.collection.admin_lists.all()]
```

Comment:
I think you can do this: `video.collection.admin_lists.values_list('name', flat=True)`

Reply:
thanks